components/nif/nifkey.hpp

```diff
@@ -50,6 +50,12 @@ struct KeyMapT {
 
     KeyMapT() : mInterpolationType(sLinearInterpolation) {}
 
+    // For serializer.
```

Comment:
In several cases I moved the private keyword.   Long term I presume that the preferred solution will be const getters for members that used to be private?  Also there's the issue of whether to create getters and setters in places where we're not 100% required to have them for consistency.